core.js

```diff
@@ -763,24 +763,34 @@ function NewCharacter(){
 
 }
 
+charQty = 1;
+for (let j = 0; j < process.argv.length; j++) {    //loops all arguments 
+  if (process.argv[j] == '-n') {                   //check for the argument -n, if 'true' user wants several chars
+    var isnum = /^\d+$/.test(process.argv[j+1]); //checks if the next argument is a number
```

Comment:
Hi @KingFruit85, I did this to avoid decimal numbers. As in the code below: 

>   //------------------------
>   if (isNaN('3.5')) { console.log ('NaN'); }
>   else {console.log ('number'); }  
>   // OUTPUT: "number"
>   //------------------------
>   if ( /^\d+$/.test('3.5') ) { console.log ('integer'); }
>   else { console.log ('not integer');} 
>   // OUTPUT: > "not integer"
>   //------------------------

Reply:
OK understood 😊